web-interface/vm-consoles-modules/vm-console-web.adoc

```diff
@@ -13,3 +13,6 @@ The *Send Key* button lists key combinations to send to the virtual machine.
 .Options for the *Serial Console*
 Use the *Disconnect* button to manually disconnect the *Serial Console* session from the virtual machine. +
 Use the *Reconnect* button to manually open a *Serial Console* session to the virtual machine.
+
+.Options for the *Desktop Viewer*
+The *Network Interface* drop-down allows you to choose the network interface for the VM you want to connect.
```

Comment:
"for the VM you want to connect." -> "for the VM to which you want to connect."

Reply:
fixed